model/BUILD.bazel

```diff
@@ -0,0 +1,163 @@
+# TODO: Rename file to 'BUILD' post-Gradle. See https://github.com/oppia/oppia-android/issues/1532.
+'''In Bazel, proto files are built using the proto_library() and java_lite_proto_library() rules.
```

Comment:
Nit change: You can write this as 
```
'''
In Bazel, proto files are built using the proto_library() and java_lite_proto_library() rules.
The proto_library() rule creates a proto file library to be used in multiple languages.
The java_lite_proto_library() rule takes in a proto_library target and generates java code.
'''
```
And add a single line space below.

Reply:
Done!